README.md

```diff
@@ -157,6 +159,22 @@ Check [fuzion-lang.dev](https://fuzion-lang.dev) for language and implementation
 - GNU make
 - libgc
 
+### MacOS
+
+> This command should install all requirements:
```

Comment:
Independent of this patch, but also affecting the other sections: `requirements` is not correct here, should be something like `required packages` or similar. Can you change this in a separate PR?

Reply:
@fridis maybe: dependencies?